components/Hero/Hero.scss

```diff
@@ -0,0 +1,128 @@
+.hero {
+  width: 100%;
+  height: 100vh;
+  background-image: url("../../public/images/hero-bg-mobile.gif");
+  background-size: cover;
+
+  &__wrapper {
+    position: relative;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    height: 100vh;
+  }
+
+  &__content {
+    position: relative;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    gap: 4px;
+
+    @include tablet-xl {
+      gap: 8px;
+    }
+  }
+
+  &__image {
+    width: 147px;
+    height: 59px;
+
+    img {
+      width: 100%;
+      height: 100%;
+    }
+
+    @include tablet {
+      position: absolute;
+      right: -127px;
+      top: -107px;
+      width: 189px;
+      height: 119px;
+    }
+
+    @include tablet-xl {
+      right: -159px;
+      top: -99px;
+      width: 163px;
+      height: 103px;
+    }
+
+    @include desktop {
+      right: -132px;
+      top: -97px;
+      width: 182px;
+      height: 115px;
+    }
+
+    @include desktop-xl {
+      right: -245px;
+      top: -205px;
+      width: 340px;
+      height: 212px;
+    }
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 28px;
+    text-align: center;
+
+    &--bold {
+      font-size: 28px;
+      line-height: 1.3;
```

Comment:
here you can inherit the font size from __title. It seems that line-height does not only apply to the modifier